diagnostic.md

```diff
@@ -6,70 +6,69 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books#index'
+resources :books, only: [:index]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It activates the migration files you have. It is going to change the schema on of project.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+Controller is part of what is created with scaffold. Scaffold is going to generate everything you need on the MVC structure. (Model, View and Controller).
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+To make tables communicate with themselves. The serializer provides the id necessary to find the relationship (one-to-one, one-to-many) between information in 2 different tables.
```

Comment:
I would say that the serializer "allows you to filter certain data out of your response"